src/views/errors/ErrorDetails.vue

```diff
@@ -1,54 +1,83 @@
 <template>
-  <section class="section">
-    <div class="columns is-mobile is-multiline">
-      <div class="column is-min-width-fit-content">
+  <section class="error-single">
+    <div class="columns is-desktop is-multiline error-header">
```

Comment:
Seria interessante colocar a classe is-vcentered

Reply:
aqui eu retirei, porque é melhor deixar as informações laterais no topo, senão dependendo do tamanho do conteúdo da esquerda, essa informação vai ficar muito abaixo na tela

Reply:
Entendi